os-server/src/main/java/app/openschool/usermanagement/api/dto/UserRegistrationHttpResponse.java

```diff
@@ -0,0 +1,62 @@
+package app.openschool.usermanagement.api.dto;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Map;
+import org.springframework.http.HttpStatus;
+
+public class UserRegistrationHttpResponse {
+
+  private String timeStamp;
```

Comment:
what is the purpose of this field

Reply:
I took it from some examples. To be honest, I didn't think about it deeply. I thought maybe this could serve as some sort of metric as it determines the server's time when sending a response. Should I avoid it too?